CHANGELOG.md

```diff
@@ -33,6 +33,8 @@ For details about compatibility between different releases, see the **Commitment
 
 ### Removed
 
+- Experimental support for BasicStation GPS timestamps which use the wrong precision (milliseconds instead of microseconds). Introduced in release `3.16.1`.
```

Comment:
Also here please us `LoRa Basics Station` (that's the trademark name).

We also should mention that users should ensure that they've upgraded their gateways to the latest firmware.